src/client.ts

```diff
@@ -1063,4 +1066,35 @@ export class SpvWalletClient {
     }
     return validateTotpForContact(this.xPrivKey, contact, passcode, requesterPaymail, period, digits);
   }
+
+  /**
+   * Syncs merkleroots from the client db to the last known block by SPV-Wallet
+   *
+   * @param {Repository} repo - Repository interface capable of reading lastEvaluatedKey and saving to the database
+   * @returns void
+   */
+  async SyncMerkleRoots(repo: Repository) {
+    let lastEvaluatedKey = await repo.getLastEvaluatedKey();
+    let requestPath = 'merkleroots';
+    let lastEvaluatedKeyQuery = '';
+
+    if (lastEvaluatedKey) {
+      lastEvaluatedKeyQuery = `?lastEvaluatedKey=${lastEvaluatedKey}`;
+    }
```

Comment:
We could add it if you'd like but I don't think it would offer any protection since merkleroots are hex encoded strings, so there shouldn't be any special characters, unless somebody's database has some malformed data. But i think it's unlikely, but I can change it here. Let me know

Reply:
It's a minor - just do as you think :) It was just some thought 🙈